exercises/concept/lucky-numbers/.docs/instructions.md

```diff
@@ -1,21 +1,41 @@
 # Instructions
 
-## 1. Find the sum of two arrays
+Your friend Kojo is a big fan of numbers.
+He has a small website for playing around with numbers: www.fun-with-numbers.com.
```

Comment:
I can't believe such a domain doesn't exist 😮

Reply:
same, maybe someone who does the exercise will by it and implement the games :smile: